### DISCUSSION TITLE
Duda output heap market -  test 1

### DISCUSSION THREAD

**Discussion by Monti7475** [IIC2133-PUC/2024-1#85] [Tarea 1] [answered]
En el test uno, en las primeras líneas
![image](https://github.com/IIC2133-PUC/2024-1/assets/130416325/96c132af-5a45-4c9b-93ea-4efa54a5f948)
No debería ser 34:15 en vez de 34:12?
(Dado que el monto de en el que se ejecuta la transacción es el monto de compra, según entendí del enunciado)
O es que entendí mal el enunciado?

**Comment by TomasKetterer** [comment, marked as answer]
Holaa, no soy ayudante pero en el enunciado dice esto:
![imagen_2024-04-19_141418424](https://github.com/IIC2133-PUC/2024-1/assets/127428983/ea963f8d-73d6-44da-98ad-99ed94a2e949)
Entonces en caso de que la última acción haya sido una venta, pones el monto de venta y en caso de que haya sido compra pones el monto de compra. Una cosa importante es que sí hay que seguir este formato:
![imagen_2024-04-19_141519811](https://github.com/IIC2133-PUC/2024-1/assets/127428983/ffdd3b50-160e-4dfe-8cca-46fc10561104)
Osea que siempre es venta.nCuenta -> compra.nCuenta y lo que cambia es el monto dependiendo de cuál haya sido la última acción

**Reply by Monti7475** [reply]
Gracias!!
